api/src/data/keys.yml

```diff
@@ -9,6 +9,7 @@ age:
   - range_more_than_65
 
 years_of_experience:
+  - no_answer
```

Comment:
I'm not sure we actually need to add it though, or maybe only at the GraphQL level… at least I don't think we use those keys in the pipeline? I'll double check.

Reply:
Previously

> This is a good start! But it's missing a key feature, which is that the `no_answer` key should be added to _buckets_, not just _facets_.
> 

Generic pipeline updated. Added "no_answer" key to the buckets section. Also, when no facets is selected, got results as you wrote before.

Reply:
> Because every field will need a `no_answer` key I don't think it makes sense to add it here, it's probably better to add it directly in `generic.ts` somewhere.

Yes, I absolutely agree. If 'no_answer' key is needed for every field, better to add it in generic.ts or at the GraphQL level (if keys.yml are not used in pipeline). I have checked - don't found straight use of keys in Mongo aggregation, but you better check.